VALUES.md

```diff
@@ -53,6 +53,22 @@ Supported forms
   As for global values, it is possible to indicate the access path by
   using `[@js.scope]` attributes on englobing modules (see below).
 
+  When the global object is itself an object constructor, the `[@@js.create]`
+  attribute may be used to instantiate it.
+
+  For instance,
+  ```ocaml
+    module[@js.scope "JavascriptClassName"] C : sig
+      val create: T1 -> ... -> Tn -> t [@js.create]
```

Comment:
> Would it make sense to have this be [@js.new], with no payload? I guess the need to introduce a new attribute name js.create is to be able to tell gen_js_api where to pick the name from (value name or current global), right?

In the "create" case, there is no name ! The global object is the constructor. 

Let's say you want to bind:
```js 
class Point {
  constructor(x, y) {
    this.x = x;
    this.y = y;
  }

  norm() {
    return Math.hypot(x, y);
  }

  static distance(a, b) {
    const dx = a.x - b.x;
    const dy = a.y - b.y;

    return Math.hypot(dx, dy);
  }
}
```
```ocaml
module[@js.scope "Point"] Point : sig 
  type t 
  val t_to_js: t -> Ojs.t
  val t_of_js: Ojs.t -> t 

  val new_point: float -> float -> t [@@js.create] (* no way to bind the constructor using [@js.new] under the scope *)
  val distance: t -> t -> float [@@js.global] 
  
  val norm: t -> float [@@js.call]
end 
```

If we were to use `[@@js.new]` without payload, it would be the same as [@@js.new "Point"] (which under the scope "Point" would be "Point.Point").

The problem with "scoped_new" is that it could imply that "new" is "unscoped". We could use the empty string "" instead of no payload (`[@@js.new ""]` ?) Or `[@@js.new_self]` ?